docs/index.asciidoc

```diff
@@ -608,12 +610,12 @@ option when authenticating to the Elasticsearch server. If set to an
 empty string authentication will be disabled.
 
 
-[id="plugins-{type}s-{plugin}-deprecated-options"]
-==== Elasticsearch Input deprecated configuration options
+[id="plugins-{type}s-{plugin}-removed-options"]
```

Comment:
For the docs update, let's follow the example in https://github.com/logstash-plugins/logstash-output-http/commit/2fae93e26b103cf9e7c0ec0d257be903070805b9#diff-cae5619b3d18ec99c5ccd0a9f6de0c6d3f53343c64692444551a7d29da6863e7

ie, remove all inline references to deprecated settings, and add a dedicated section at the bottom, above "common options", with a link to that section just before the config table.

cc @karenzone